docs/source/introduction/introduction.rst

```diff
@@ -1,72 +1,120 @@
+	* HTML formatted log support. 
+	* Test time tracking with millisecond accuracy.
+	* Separate log file and reports per test suite.
+
+* Automated professional looking Extent report generation.
+* Automated test script generation.
+* Automated test plan generation.
+* Automated test plan formatting for BDD based test plan.
+* GUI test selector for debugging, selective testing and avoids user error. 
+* TestCase and TestUnit filters based on groups.
+* Exception checking.
+* Known to fail test case support.
+* DataProvider support.
+* Parallel testing support.
+
+	* Execute multiple test suites at the same time.
+
+* Sequentialize or skip test cases using test script, test list or test annotation.
+* Global parameter support.
+* Stop on fail support.
+* Properties based test framework configuration.
+* Failure highlight for easy debugging. 
+
+	* Failed test cases are highlighted on console logs
+	* Failed test cases are reported with bug reference number for easy tracking.
+
+* Listeners for application development. 
+
 Framework Glossary
 ##################
 
 .. csv-table:: 
-	:header: Keyword, Description
-	:widths: 30, 70
-	:stub-columns: 0
-	
-	Test Suite, A collection of test cases that are designed specifically to test the system under test
-	Test Runner, A class which is the entry point to a test application. It is responsible for running and tracking test cases from the start to end
-	Test Case, A class which contains set of instructions that will be performed on the system under test
-	Test Unit, A method within a test case that represents the smallest and independent executable unit
-	Test Script, A set of instructions to guide the test runner on how to execute test cases. The test script is represented by xml script
-	Scan Scope, A section of the Java project which will be scanned during the search of test cases
-	Test Status, "A state of a test case at the line of execution (PASS, FAIL, SKIP or KTF)"
-	Unit Outcome, "The final outcome of the test unit (PASS, FAIL, SKIP or KTF)"
-	Test Outcome, "The final outcome of a test case (PASS, FAIL, SKIP or KTF)"
+   :header: Keyword, Description
+   :widths: 30, 70
+   :stub-columns: 0
+   
+   Test Suite, A collection of test cases that are designed specifically to test the system under test
+   Test Runner, A class which is the entry point to a test application. It is responsible for running and tracking test cases from the start to end
+   Test Case, A class which contains set of instructions that will be performed on the system under test
+   Test Unit, A method within a test case that represents the smallest and independent executable unit
+   Test Script, A set of instructions to guide the test runner on how to execute test cases. The test script is represented by xml script
+   Scan Scope, A section of the Java project which will be scanned during the search of test cases
+   Test Status, The state of a test case at the time of execution (namely: PASS, FAIL, SKIP or KTF)
```

Comment:
This will not compile because it needs quotes around text, otherwise , is considered as separate column